src/main/java/ch/njol/skript/expressions/ExprName.java

```diff
@@ -97,336 +96,151 @@
 	static final MethodHandle TITLE_METHOD;
 
 	static {
+		register(ExprName.class, String.class, "(1¦name[s]|2¦(display|nick|chat|custom)[ ]name[s])", "players/entities/itemtypes/inventories/slots");

```

Comment:
Slots are convertible to itemtypes, you may test whether removing slots from the accepted types causes any issues (changers primarily).

Reply:
Tested this. Works with the getter but not the changer, presumably because you have to update the slot's item(?).